pages/mesosphere/dcos/services/kafka/2.7.0-2.3.0/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layout.pug
+navigationTitle: Kafka 2.7.0-2.3.0
+excerpt:
+title: Kafka 2.7.0-2.3.0
+menuWeight: 1
```

Comment:
Since this is now the latest major version, can you delete the entire folders of  Kafka `2.5.0-2.1.0` and `2.6.0-2.2.1`? That way, the only versions that show up in the service docs are `2.3.0` and `1.1.1` (which is major version -1)